app/controllers/posts_controller.rb

```diff
@@ -36,13 +43,29 @@ def create
 
   # PATCH/PUT /posts/1 or /posts/1.json
   def update
-    respond_to do |format|
-      if @post.update(post_params)
-        format.html { redirect_to post_url(@post), notice: "Post was successfully updated." }
-        format.json { render :show, status: :ok, location: @post }
-      else
-        format.html { render :edit, status: :unprocessable_entity }
-        format.json { render json: @post.errors, status: :unprocessable_entity }
+    if params[:post][:message].present?
+      time = Time.new
+      puts(params)
+      create_comment = {
+        :message => params[:post][:message],
+        :commentId => SecureRandom.uuid,
+        :postId => @post.postId,
+        :posterId => current_user.studentId,
+        :commentDate => time.strftime("%Y-%m-%d"),
+      }
+      Comment.create(create_comment)
```

Comment:
Does this create a new comment with the same postId or does it just update the same comment that is being edited?

Reply:
It creates a new comment with the postId set to the post its commented under